### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4287] [General] [answered]
I've ported my projects from toggl to kimai and used it 2 month now.
Today I've updated to recent version of kimai and just want to say `Thank you`. 
Upgrade was smooth and straight forward and everything work as expected. Also daily work with the timetracker-software is easy and straight forward.
**Great work!** 🏆 

**BTW:**
 Would be nice to have a link to the current CHANGELOG.md (https://github.com/kimai/kimai/releases) straight from the "About Kimai" page. 
This way Admin can see what's new in other versions easily from the kimai UI.

Maybe just a link from the Version-No. ?
![image](https://github.com/kimai/kimai/assets/1778068/a9139acd-6c68-4366-babb-20ac93addf14)

**Comment by kevinpapst** [comment, marked as answer]
The doctor screen should show you new releases.
But the result is cached for 24 hours... so you might have to wait after I published a new version
